src/content/blog/drupalcon-portland-wrapup.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Drupalcon Portland 2024 Wrapup"
+pubDate: 2024-05-14
+# modifiedDate: 2024-04-23
+summary: Drupalcon Portland 2024 Wrapup
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/05/platform.sh-booth-with-ddev-block.png
+  alt: Platform.sh Booth With "Proud Sponsor of DDEV" block
+categories:
+  - Community
+---
+
+We go to Drupalcons because they're a great way to stay in touch with one of our valued communities. DDEV has deep roots in the Drupal world, and it's always a delight to see our stakeholders at Drupalcon and hear their successes and frictions.
```

Comment:
```suggestion
We go to DrupalCon because it's a great way to stay in touch with one of our valued communities. DDEV has deep roots in the Drupal world, and it's always a delight to see our stakeholders at DrupalCon and hear their successes and frictions.
```